src/clib/pio_file.c

```diff
@@ -429,7 +429,7 @@ int PIOc_deletefile(const int iosysid, const char filename[])
         MPI_Barrier(ios->io_comm);
     }
 
-    //   Special case - always broadcast the return from the
+    /* Special case - always broadcast the return from the ??? */
```

Comment:
Always broadcast the return from the delete command, I'm not entirely sure why at this point.